jenkins/slave-base/Dockerfile.centos7

```diff
@@ -14,13 +14,9 @@ RUN yum -y install gnutls-utils
 RUN echo ${APP_DNS}
 RUN gnutls-cli --insecure --print-cert ${APP_DNS} </dev/null| sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' > /tmp/oc_app.crt
 
-# Install Sonar Scanner
```

Comment:
Is there a reason you removed the sonar cli installation?

Reply:
@tjaeschke  - this was double in .. :) check the source code ...